TestingHelpers/MockFile.cs

```diff
@@ -13,6 +13,8 @@ public class MockFile : FileBase
         readonly IMockFileDataAccessor mockFileDataAccessor;
         readonly MockPath mockPath;
 
+        public static readonly Encoding DefaultEncoding = new UTF8Encoding(false, true);
```

Comment:
There's already `MockFileData.DefaultEncoding`. See usage in `AppendAllText` already.